README.md

```diff
@@ -15,6 +15,7 @@ const Client = require('ilp').Client
 
 const client = new Client({
   type: 'bells',
+  prefix: 'red-ilpdemo',
```

Comment:
I think `ilpdemo.red` is nicer than `red-ilpdemo`. Elements of ILP addresses don't have to be ledgers, they can be purely structural, i.e. to group related ledgers together.